__mocks__/POST.mock

```diff
@@ -0,0 +1,17 @@
+HTTP/1.1 200 OK
+Content-Type: application/json
+{{#code}}
+(() => {
+    const jwtToken = "mocked_jwt_token";
+
+    return {
+        status: 200,
+        headers: {
+            'X-Requested-By': 'register/user'
```

Comment:
not sure if we need this header here at all.
even if we do, here is an example of how it might look like:
https://github.com/testinggospels/camouflage/blob/develop/mocks/users/GET.mock